README.md

```diff
@@ -69,11 +72,29 @@ You can also choose one of the presets provided by the ROS2 Gem to create a LiDA
 the **RUNTIME** requirements.*
 
 ## Setup
+
 1. **Clone the Gem's repository.**
     ```bash
     git clone https://github.com/RobotecAI/o3de-rgl-gem.git
     ```
-2. **Register the Gem.** \
+2. **Switch to the correct branch.** \
+   Depending on which version of O3DE and ROS2 gem you use this step may differ.
+   Below is a universal table of supported build combinations:
+
+   | o3de-rgl-gem | [o3de/o3de-extras](https://github.com/o3de/o3de-extras) | [Robotecai/o3de-extras](https://github.com/robotecai/o3de-extras) | [o3de](https://github.com/o3de/o3de) |
+   |--------------|---------------------------------------------------------|-------------------------------------------------------------------|--------------------------------------|
+   | development  | development                                             | N/A                                                               | development      |
+   | development  | development                                             | N/A                                                               | stabilization/2409      |
+   | development  | N/A                                                     | rgl/stable-dev                                                    | point-release/23103                  |
+   | main         | point-release/23103                                    | N/A                                                               | point-release/23103                  |
```

Comment:
Why o3de-rgl-gem/main for o3de-extras is N/A?

Reply:
In the table there are two possible o3de-extras repos: The main o3de/o3de-extras repo and the robotecai/o3de-extras fork. I'm not sure if you've seen the rendered markdown table but it looks like this:
![image](https://github.com/user-attachments/assets/79b65de2-c18e-43dc-b7d5-4692b7be1b0f)
As you can see it features the two forks so in some combinations one of the forks is not applicable.